src/robusta/core/sinks/telegram/telegram_sink.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
+from telethon import TelegramClient
+from telethon.tl.types import DocumentAttributeFilename
+
+from .telegram_sink_params import TelegramSinkConfigWrapper
+from ..transformer import Transformer
+from ...reporting.blocks import (
+    MarkdownBlock,
+    TableBlock,
+    FileBlock,
+)
+from ...reporting.base import (
+    Finding,
+    FindingSeverity, BaseBlock,
+)
+from ..sink_base import SinkBase
+from ...reporting.utils import convert_svg_to_png
+
+SEVERITY_EMOJI_MAP = {
+    FindingSeverity.INFO: u"\U0001F7E2",
+    FindingSeverity.LOW: u"\U0001F7E1",
+    FindingSeverity.MEDIUM: u"\U0001F7E0",
+    FindingSeverity.HIGH: u"\U0001F534",
+}
+INVESTIGATE_ICON = u"\U0001F50E"
+
+
+class TelegramSink(SinkBase):
+    def __init__(self, sink_config: TelegramSinkConfigWrapper, cluster_name: str):
+        super().__init__(sink_config.telegram_sink)
+        self.cluster_name = cluster_name
+        self.api_id = sink_config.telegram_sink.api_id
+        self.api_hash = sink_config.telegram_sink.api_hash
+        self.bot_token = sink_config.telegram_sink.bot_token
+        try:  # if the recipient is a group, the group id should be provided, and passed as int
+            self.recipient = int(sink_config.telegram_sink.recipient)
+        except ValueError:
+            self.recipient = sink_config.telegram_sink.recipient
+
+        self.send_files = sink_config.telegram_sink.send_files
+
+    def write_finding(self, finding: Finding, platform_enabled: bool):
+        asyncio.run(self.__send_telegram_message(finding, platform_enabled))
+
+    async def __send_telegram_message(self, finding: Finding, platform_enabled: bool):
+        client = TelegramClient('Robusta', self.api_id, self.api_hash)
+        await client.start(bot_token=self.bot_token)
+        async with client:
+            await client.send_message(
```

Comment:
this isnt in a try catch, is this on purpose?

Reply:
auto tear down the client.

Reply:
we dont want to print anything extra or the stack trace to the logger??

Reply:
how do the errors from send message look in the log / stack trace? are they readable to someone unfamiliar with the plugin?

Reply:
If they have an error it's printed to the log as error.
If an exception is thrown it's being caught on a higher level, so that the finding will be send to the other sinks